tests/pyspark/test_pyspark_config.py

```diff
@@ -326,3 +335,25 @@ class TestSchema(DataFrameModel):
             dict(output_dataframemodel_df.pandera.errors["SCHEMA"])
             == expected_dataframemodel["SCHEMA"]
         )
+
+    @pytest.mark.parametrize("cache_enabled", [True, False])
+    @pytest.mark.parametrize("unpersist_enabled", [True, False])
+    # pylint:disable=too-many-locals
+    def test_pyspark_cache_settings(
+        self,
+        cache_enabled,
+        unpersist_enabled,
+    ):
+        """This function validates that caching/unpersisting works as expected."""
```

Comment:
Update this docstring to say that it's actually testing the config setter and getters

Reply:
Done, tks